src/test/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtilTest.java

```diff
@@ -77,13 +77,13 @@ public void testMetadataFilesGenerationAllFiles(@SystemOutGuard.SysOut Capturabl
                 System.out, "checks", "filters", "filefilters");
 
         final String[] expectedErrorMessages = {
-            "31: " + getCheckMessage(MSG_DESC_MISSING, "AbstractSuperCheck"),
```

Comment:
Lines got moved 1 row down due to the annotation.
![Screenshot from 2023-06-01 21-07-40](https://github.com/checkstyle/checkstyle/assets/23459549/8ec1f939-ecce-4c7a-b6aa-1f029dd33155)

Reply:
@stoyanK7 , please try to share text representation of errors.